### ISSUE TITLE
Better Numbering System

### ISSUE THREAD

**Issue by jalanmiddleton** [DeveloperLiberationFront/Excel-Function-Visualizer#161]
From #156 

> Section III.A: I do not understand why the goals are numbered “1”, “!1”, etc.

**Comment by CaptainEmerson** [comment]
I agree that it was confusing. Rather than explaining it, I'd just turn the negative ones into positives.
